### ISSUE TITLE
Make Dailies Progress World Specific

### ISSUE THREAD

**Issue by frodare** [ToroCraft/Dailies#51]
> First of all, I want to said, I really enjoy the mod.
>  
> I was wondering if there is a way to reset the dailies for new worlds?
> I play 3 or 4 different world at the same time and all the dailies are the same for all the world to include the progress on each.

~Syotos01

**Comment by frodare** [comment]
I think this would be a good idea too.  Not sure how much would be involved on the service side though ... @pandapaul .  We would need to store the progress under the player's ID concatenated with a server settable ID, and of course make it backwards compatible so that if no config is set it uses the "global" progress.

**Comment by pandapaul** [comment]
Oof.  Would be a lot of work.  Off the top of my head, things to consider are the service storage layer concerns.  Maybe take a look at what gets stored there, @frodare, and we can go from there.  It could be cool.  It will just be tough.

**Comment by frodare** [comment]
What if we used a set delimiter like `~!π!~` to join username and server code together and then treated like it was a different user?  Probably not the most elegant solution, but it seems like it might be a simple way to accomplish this.

**Comment by pandapaul** [comment]
Yeah.  Then we'd be establishing a custom user identifier.  That might be fine, but I wouldn't want to overload the username property with that.  I added you as a collaborator on the Heroku side of things for both the staging and production apps.  Hopefully that'll give you access to the database.  Models are here:  https://github.com/pandapaul/minecraft-dailies-server/tree/master/db

I don't mind your proposal since basically everywhere we key off username we'll need to key off of a combined username + server id.  I'd just rather that be stored more explicitly.

**Comment by frodare** [comment]
That would be better for sure.
